templates/Azure-Blob-Auto-Expiration.json

```diff
@@ -0,0 +1,132 @@
+{
+  "id": "/providers/Microsoft.Logic/galleries/public/templates/666271ff871243e59b70460a9c2056db",
+  "type": "Microsoft.Logic/galleries/templates",
+  "name": "666271ff871243e59b70460a9c2056db",
+  "properties": {
+    "author": {
+      "displayName": "Yuan Zheng"
+    },
+    "categoryNames": [
+      "schedule"
+    ],
+    "description": "Once a day, scan a storage account and expire all blobs older than defined value.",
+    "displayName": "Scheduler - Azure Blob Storage TTL",
+
+    "definition": {
+      "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+      "actions": {
+        "For_each": {
+          "actions": {
+            "Condition": {
+              "actions": {
+                "Delete_blob": {
+                  "description": "If blob is older than the expiration age, delete it",
+                  "inputs": {
+                    "host": {
+                      "connection": {
+                        "name": "@parameters('$connections')['azureblob']['connectionId']"
+                      }
+                    },
+                    "method": "delete",
+                    "path": "/datasets/default/files/@{encodeURIComponent(encodeURIComponent(items('For_each')?['Id']))}"
+                  },
+                  "runAfter": {},
+                  "type": "ApiConnection"
+                }
+              },
+              "description": "Check LastModified timestamp and whether older than the expiration age variable",
+              "expression": "@less(items('For_each')?['LastModified'], addDays(utcnow(), variables('ExpirationAgeInDays')))",
+              "runAfter": {},
+              "type": "If"
+            }
+          },
+          "description": "Scan all blobs in this folder",
+          "foreach": "@body('List_blobs')?['value']",
+          "runAfter": {
+            "List_blobs": [
+              "Succeeded"
+            ]
+          },
+          "type": "Foreach"
+        },
+        "List_blobs": {
+          "inputs": {
+            "host": {
+              "connection": {
+                "name": "@parameters('$connections')['azureblob']['connectionId']"
+              }
+            },
+            "method": "get",
+            "path": "/datasets/default/foldersV2/@{encodeURIComponent(encodeURIComponent('L3Rlc3Q='))}",
```

Comment:
I think the value should be `/datasets/default/foldersV2/`, without the url encoding part?

Reply:
fixed